ceph/init.sls

```diff
@@ -10,6 +10,10 @@ install_ceph_pkgs:
   pkg.installed:
     - pkgs: {{ settings.packages }}
 
+create_ceph_config_file:
+  file.touch:
```

Comment:
Yes I know
Fix in next PR https://github.com/stamak/ceph-formula/commit/e1e1cb7672bd39a811f15416bb144aa86b2f7635